test/spec/features/outline/MultiSelectionOutlineSpec.js

```diff
@@ -0,0 +1,74 @@
+
+import {
+  bootstrapDiagram,
+  inject,
+} from 'test/TestHelper';
+
+import selectionModule from 'lib/features/selection';
+
+import {
+  classes as domClasses,
+  query as domQuery
+} from 'min-dom';
+
+describe('features/outline/MultiSelectionOutline', function() {
+
+  beforeEach(bootstrapDiagram({ modules: [ selectionModule ] }));
+
+  describe('bootstrap', function() {
+
+    beforeEach(bootstrapDiagram({ modules: [ selectionModule ] }));
+
+    it('should bootstrap diagram with component', inject(function() {
+
```

Comment:
What is the purpose of this empty test?

Reply:
I followed what is done in `test/spec/features/selection/SelectionVisualsSpec.js` L#29

Reply:
We don't need that spec here, but I'd expect that such spec exists in the `OutlineSpec`, which we also need to update, cf. https://github.com/bpmn-io/diagram-js/blob/39e138c092026f839d3f417410486ed33a76cd1e/test/spec/features/outline/OutlineSpec.js#L6.

Outline spec should pull in the outline, not the selection module, and, cf. https://github.com/bpmn-io/diagram-js/pull/943#discussion_r1824086422, this would make the feature available.

Reply:
I'll help you with the changes!

Reply:
Cf. https://github.com/bpmn-io/diagram-js/pull/943/commits/ef8528e1116f5100329c375df8222eaf775f739a where I adjusted this.

Reply:
Essentially when you dropped the dependency `selection -> outline` we missed on adjusting the tests accordingly.

Reply:
yeah, I barely touched the tests, I wanted to first make sure that I go in a good direction with the main changes